app/controllers/application_controller.rb

```diff
@@ -107,9 +107,11 @@ def source
     session[:source]
   end
 
+  # Allow session[:source] to re-set on every interaction so we can track all
+  # unique source entry points in Mixpanel.
```

Comment:
This may end up changing to only set once and not override, but for now will revert to how it was originally set up.